general/development/policies/accessibility.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Accessibility
+tags: []
+---
+
+import clsx from 'clsx';
+
+Moodle is designed to provide equal functionality and information to all people. This means that there should be no barriers for people regardless of disabilities, assistive technologies that are used, different screen sizes and different input devices (for example mouse, keyboard and touchscreen).
+
+## Accessibility conformance
+
+As part of our ongoing commitment to provide an accessible platform and to continuously improve compliance with WCAG 2.1 Level AA, Moodle has been working with an external auditor to review the platform since January 2020.
+
+Key pages within Moodle, which are representative of its overall accessibility and functionality, are audited using both automated tools and user journey testing. The WCAG 2.1 Level A and Level AA issues raised from this audit were all addressed in Moodle 3.10 and Moodle 3.9.3 (both released on November 9th 2020). Many of these [accessibility improvements](https://tracker.moodle.org/browse/MDL-67688) where also back-ported to Moodle 3.7.5+ and Moodle 3.8.2+, where possible.
+
+:::info Moodle's accreditations
+
+Both Moodle, and the Moodle Mobile App hold a WCAG 2.1 AA accreditation based on the successful resolution of issues identified during audits.
+
+These accreditations are renewed each year.
+
+<div className={ clsx( 'container' ) } >
```

Comment:
What's the benefit of using clsx here? As there are no conditionals, why you haven't used class="container" (which doesn't require importing clsx)?